distro/adaptation/amazon_linux

```diff
@@ -0,0 +1,5 @@
+apache2: httpd
+apt-file:
+arping: iputils
```

Comment:
this one is not in amazon_linux-2, so it should not be put in common file. Nothing need to be done for this arping package.